lighthouse-core/gather/gatherers/iframe-elements.js

```diff
@@ -18,12 +20,11 @@ function collectIFrameElements() {
   // @ts-expect-error - put into scope via stringification
   const iFrameElements = getElementsInDocument('iframe'); // eslint-disable-line no-undef
   return iFrameElements.map(/** @param {HTMLIFrameElement} node */ (node) => {
-    const clientRect = node.getBoundingClientRect();
```

Comment:
This gatherer's output is part of our "public artifacts" which we expose to plugins and promise to not change within a major version, so we'll want to ensure backwards compatibility (at least until LH 7) for those.

https://github.com/GoogleChrome/lighthouse/blob/fe74339f1ee2b5606d5a7af19b6b52bd622e862a/types/artifacts.d.ts#L61-L65

Reply:
Good call, given it directly calls the `getBoundingClientRect()` function, it's probably better to keep that clientRect anyways 👍